content/configuration/index.md

```diff
@@ -414,7 +415,16 @@ module.exports = {
   },
 
   [node](node): {
-    /* TODO */
+    // Polyfills and mocks to run Node.js-
+    // environment code in non-Node environments.
+    
+    [console](node#node-console): false, // boolean | "mock"
+    [global](node#node-global): true, // boolean | "mock"
+    [process](node#node-process): true, // boolean
+    [__filename](node#node-__filename): "mock", // boolean | "mock"
+    [__dirname](node#node-__dirname): "mock", // boolean | "mock"
+    [Buffer](node#node-buffer): true, // boolean | "mock"
+    [setImmediate](node#node-): true // boolean | "mock" | "empty"
```

Comment:
> If I'm trying to submit a PR for this in the future, is there a reference for instructions on how I can see the compiled docs to avoid taking up time for typos like this in the future? :)

Not currently, but I'm planning to do some README/CONTRIBUTING/WRITER'S GUIDE updates soon. You can run `npm run build && npm test` to see if the tests pass or `npm run build-test` to run a full build and test it using `http-server`. Note that the full build process does take a while, we don't mind if you let it run on travis -- sometimes it's nice to let it run remotely so you can continue other work in the meantime. 

Also, certain tests, like markdown linting, can be run individually without running a doing a full build but others, like link testing, do require the actual site.